src/components/SearchResult/SearchResult.story.svelte

```diff
@@ -57,6 +58,18 @@
 				proposedBillsCount: 1,
 				url: '#n'
 			}
+		],
+		promises: [
```

Comment:
The issue (#129) said that the promises should be under `bills`, but i think this make more sense?

This is what mentioned in the issue:

- You can add this mock data in the story file:
```javascript
const searchResults: SearchResults = {
	// ...
	bills: [
		{ heading: 'โครงการ 1 อำเภอ 1 ทุน (ODOS: One District One Scholarship)', promiseStatus: PromiseStatus.notStarted, url: '#m' },
		{ heading: '“รถไฟฟ้า กทม.” 20 บาทตลอดสาย', promiseStatus: PromiseStatus.inProgress, url: '#n' },
	]
}
```

Reply:
Yup, it's my typo mistake here. Should be a promise instead of the bill in the mock data. (For under I mean below bill for the UI)

You are doing it right!